.circleci/config.yml

```diff
@@ -4,56 +4,31 @@ version: 2
 jobs:
   python:
     docker:
-      - image: udata/circleci:2-alpine
+      - image: cimg/python:3.7.13
       - image: mongo:3.6
       - image: redis:alpine
     environment:
        BASH_ENV: /root/.bashrc
     steps:
       - checkout
-      - run:
-          name: Compute Python dependencies key
-          command: cat requirements/*.pip > python.deps
       - run:
           name: Get the base reference branch
           command: export BASE_BRANCH=$(base_branch)
-      - restore_cache:
-          keys:
-          - py3-cache-v9-{{ arch }}-{{ checksum "python.deps" }}
-          - py3-cache-v9-{{ arch }}-{{ .Branch }}
-          - py3-cache-v9-{{ arch }}-{{ .Environment.BASE_BRANCH }}
       - run:
           name: Install python dependencies
           command: |
-            virtualenv venv
-            source venv/bin/activate
-            pip install -r requirements/develop.pip
-            pip install -e .
-      - save_cache:
-          key: py3-cache-v9-{{ arch }}-{{ checksum "python.deps" }}
-          paths:
-          - venv
-      - save_cache:
-          key: py3-cache-v9-{{ arch }}-{{ .Branch }}
-          paths:
-          - venv
+            poetry install
       - run:
           name: Run tests
           command: |
             mkdir -p reports/python
-            source venv/bin/activate
-            inv test --report --ci
+            poetry run inv test --report --ci
```

Comment:
We could use a Makefile, as we've done for newer packages?

Reply:
I love Makefiles but I'm not sure it's easier to maintain another file on every project

Reply:
Maybe use a common package for that exposes the commands we have for invoke in every project currently?